components/fuipwahef749384h2e.yml

```diff
@@ -0,0 +1,23 @@
+name: emotional_bot_response
+display_name: Emotional Bot Response
+component_type: Generative
+model_type: NN-based
+is_customizable: true
+author: publisher@deeppavlov.ai
+description: Propose alternative hypotheses with specified emotional color.
```

Comment:
if it proposes a hypotheses, why it is a candidate annotator NOT a skill?
I suggest to make a few changes and move it to skills

Reply:
Changed description

Reply:
so, it doesn't produce a new one hypotheses, but processes existing hypothesEs?

Reply:
yes